docs/pages/platform/upgrading/2.2.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: "Upgrading to 2.2"
+  parentTitle: "Upgrading"
+  description: "Guide to upgrade to Liveblocks version 2.2"
+---
+
+We are making Thread "resolved" a first-class citizen property. You don't have
+to use thread's metadata to set a thread as resolved anymore.
+
+## All breaking changes are for Comments and only if you use a metadata "resolved"
+
+If you are using Comments without using a metadata `resolved`, there are no
+breaking changes for you! However, if you are using a thread's metadata
+"resolved", and have logic based on the webhook event `threadMetadataUpdated`,
+keep reading.
+
+### Webhook events
+
+When you upgrade to 2.2, our React component will update the thread first-class
+citizen property instead of the metadata. When a thread is marked as resolved or
```

Comment:
This sentence is flowing a bit weirdly. Did you mean to say "update to" here?

Reply:
The default behaviours of the react component when clicking on "mark as resolved" will be to update the new property instead of the metadata.